components/BaseInput.vue

```diff
@@ -54,4 +55,20 @@ const onInput = (e: Event) => {
 .input__input[type="number"] {
   width: 80%;
 }
+
+.login__email.input__input,
```

Comment:
You are right :), but it turned out that only number input has different width so just moved the width:100% definition to other class, so for now I think it will be good without the prop.